routes/tournamentRoute.js

```diff
@@ -24,8 +24,29 @@ router.put('/:id', (req, res, next) => {
     res.send("Putted in here");
 });
 
-router.get('/:id/round/:round', (req, res, next) => {
+//get all rounds of a tournament
+router.get('/:id/round', [getAllRoundsByTournamentId],async (req, res, next) => {
+    res.json(res.locals.tournament);
+});
+
+//get one round of a tournament by Id
+router.get('/:id/round/:round', [getOneRoundFromTournamentById],async (req, res, next) => {
+    res.json(res.locals.round);
+});
+
+//update round by Id
+router.put('/:id/round/:round', [updateRoundById], async (req, res, next) => {
+    res.json(res.locals.tournament);
+});
 
+//delete a round in a tournament by Id
+router.delete('/:id/round/:round', [deleteRoundById], async (req, res, next) => {
+    res.send("Delete a specific round");
```

Comment:
```suggestion
    res.sendStatus(204);
```
When deleting send Status at least